cityofaustin/atd-data-and-performance
# Update NPM packages

## steps to test

1. navigate to page. the home page, traffic camera dashboard, signal status dashboard, and signal evaluation dashboard. Make sure they work normally

This is the result of `npm update --save`. There is one moderate vulnerability remaining but i did not attempt to resolve it. NPM suggests we bump to Next v13, which would be fun :)

```
❯ npm audit
# npm audit report

postcss  <8.4.31
Severity: moderate
PostCSS line return parsing error - https://github.com/advisories/GHSA-7fh5-64p2-3v2j
fix available via `npm audit fix --force`
Will install next@13.5.4, which is a breaking change
node_modules/postcss
  next  9.3.4-canary.0 - 13.5.4-canary.11
  Depends on vulnerable versions of postcss
  node_modules/next
```
